bitnami/appsmith/templates/externaldb-secrets.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if not .Values.mongodb.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-externaldb" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
Could we add the namespace line right after name, just like all the other assets? Reference: https://github.com/bitnami/charts/blob/main/bitnami/sonarqube/templates/externaldb-secret.yaml#L6

Reply:
Done